### ISSUE TITLE
Can anyone reproduce the claimed accuracy?

### ISSUE THREAD

**Issue by zhaowt61** [jfzhang95/pytorch-deeplab-xception#144]
I trained on Resnet using Pascal + SBD, but got an accuracy of 74.5%. All settings are default values.

**Comment by zhaowt61** [comment]
By using the following settings, I managed to achieve 77.24% at 24 epochs.

Setting:
1 dataset: Pascal + SDB
2 4 gpus, BatchSize = 16
3 Backbone: ResNet
4 —freeze-bn False
5 LR: (replace the function def call(self, optimizer, i, epoch, best_pred) in utils/lr_scheduler.py)
if epoch<10:
    lr = 0.007
if epoch >= 10 and epoch <=20:
    lr = 0.0007
if epoch >= 21 and epoch <=40:
    lr = 0.00007
